build/Microsoft.DotNet.Cli.BundledSdks.props

```diff
@@ -1,6 +1,9 @@
 <Project ToolsVersion="15.0">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  
   <ItemGroup>
-    <BundledSdk Include="Microsoft.Core.Sdk" Version="1.0.0-RC2" />
-    <BundledSdk Include="Microsoft.Web.Sdk" Version="1.0.0-RC2" />
+    <!-- CLI cannot use the latest SDK until we move away from SDK PackageRef -->
```

Comment:
@dsplaisted this is where we specify the explicit version for the Microsoft.Net.Sdk that we package in the layout.